docs/methoddocs/filleeventqueries.md

```diff
@@ -1 +1,300 @@
 # File Event Queries
+
+```eval_rst
+.. autoclass:: py42.clients.file_event.FileEventClient
+    :members:
+    :show-inheritance:
+```
+
+```eval_rst
+.. autoclass:: py42.sdk.queries.fileevents.file_event_query.FileEventQuery
+    :members:
+    :show-inheritance:
+```
+
+# Filter Classes
+
+Each of the following classes can be used to filter Forensic Search results based on the class's corresponding event
```

Comment:
Is `event` attribute the `term`?

Reply:
Yeah, I wasn't sure the best way to word it. I'm not referring to the python attribute (`_term`) but the part of the event itself that the class will filter on. Maybe event "detail" is a better word? 

Just trying to tie the "class names" > "event columns in the UI" in a generic but clear way.